app/models/user.rb

```diff
@@ -1,10 +1,14 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
- [ ] Hi @GordonOtieno, I got the error on the screenshot below when I tried to sign up or sign in.

![Screenshot from 2022-12-14 07-38-43](https://user-images.githubusercontent.com/63087491/207524498-6fca56bc-b794-423a-8513-535b5fea45bf.png)

A user should be able to sign up and confirm his email